app/components/Channels/CloseChannelDialog.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Flex } from 'rebass'
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl'
+import { withFieldState } from 'informed'
+import Delete from 'components/Icon/Delete'
+import { Dialog, Text, Heading, Button, Checkbox, Form } from 'components/UI'
+import messages from './messages'
+import Overlay from '../DialogOverlay'
+
+const DialogWrapper = ({ intl, isOpen, onClose, onCancel }) => {
+  if (!isOpen) {
+    return null
+  }
+  const checkboxFieldName = 'actionACK'
+
+  // bind button disabled state to a form field or ignore it if we are dealing with a remote
```

Comment:
Reference to remote wallet can be removed from comment.

Reply:
This is still wip

Reply:
removed